### PR TITLE
Assignment1

### PR DESCRIPTION
**Pull request by tnadalini**

### CODE REVIEW

File: wordle-web/pages/game.vue [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+          <p>
+            "You saved me...why?" - Monkey
+            "mm...Monkey" - Master Oogway
+          </p>
+          <hr class="my-3" />
+        </v-card-text>
+        <v-card-actions>
+          <v-spacer />
+          <v-btn color="secondary" nuxt to="/"> home </v-btn>
+          <v-btn color="primary" nuxt to="/game"> restart </v-btn>
```
Comment (DakotaCondos) [inline wordle-web/pages/game.vue:24]: Link to the main page is there and good use of `nuxt to="/game"` to restart. Is there a link to your azure website? I don't see it anywhere

Comment (DakotaCondos) [reply wordle-web/pages/game.vue:24]: Also it looks like there are no changes to the YML file in the workflow.

File: wordle-web/pages/index.vue [hunk 2]
```diff
@@ -72,6 +72,7 @@
           </a>
         </v-card-text>
         <v-card-actions>
+          <v-btn color="primary" nuxt to="/game"> Game </v-btn>
```
Comment (tremorris1999) [inline wordle-web/pages/index.vue:75]: Links to game.vue!

**Review by tremorris1999 — CHANGES_REQUESTED** [review]
## Instructions

1. Create a Nuxt app ✅
   - Uses TypeScript ✅
   - Update Vue components to TypeScript ✅
   - Uses Vuetify ✅
   - Uses Jest ✅

3. Create a simple page called game.vue ✅
   - Page should have a card with a heading that with a clever name for the game ✅
   - some default placeholder body text ✅
   - and a button to take you back to the home page ✅
   - there should be a button the home page to access this page ✅

4. Check all the code into your repo fork ✅

5. Create a static web app in Azure ❌ (at least not in workflow or linked in some kind of readme or comment)
   - Create the container where your GitHub repo will deploy
   - Link the site with your GitHub repo

6. Update the CI/CD pipeline ❌ (not shown as PR is right now)
   - Pipeline runs on each check in
   - Jest tests run and pass ❌
   - Nuxt app builds ❌
   - Nuxt app deploys to Azure ❌

7. Azure site works ❌ (needs a link at least)
   - Post your Azure URL in your readme.md file. Replace the URL in the readme with your URL. ❌

## Extra Credit

- Come up with up to three unique ideas for stories we could implement as part of our project. Post these in your readme.md. ❌
- One point per idea up to three points

**Comment by mmwoodfo** [comment]
In the future, just make sure that you are making a separate branch for your assignment, rather than making a PR using your main. Code changes showing up here in the PR should only be relevant to the assignment the PR is for.

Here is an example of a PR that uses an Assignment 1 branch: https://github.com/IntelliTect-Samples/EWU-CSCD379-2022-Spring/pull/11

Great job!

File: wordle-web/pages/game.vue [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<template>
+  <v-row justify="center" align="center">
+    <v-col cols="12" sm="8" md="6">
+      <v-card-title class="headline justify-center">
+        <h1>Monke</h1>
+      </v-card-title>
```
Comment (mmwoodfo) [inline wordle-web/pages/game.vue:6]:
```suggestion
      <v-card-title class="headline text-h1 justify-center">
        Monke
      </v-card-title>
```

Consider using the vuetify classes for setting text heading sizes: https://vuetifyjs.com/en/styles/text-and-typography/

File: wordle-web/pages/game.vue [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      </v-card>
+      <v-card>
+        <v-card-title class="headline">
+          Welcome to the game page! :)
+        </v-card-title>
+        <v-card-text>
+          <p>
+            "You saved me...why?" - Monkey
+            "mm...Monkey" - Master Oogway
+          </p>
```
Comment (mmwoodfo) [inline wordle-web/pages/game.vue:18]:
```suggestion
            "You saved me...why?" - Monkey
            "mm...Monkey" - Master Oogway
```

You don't need to wrap this in a `<p>` tag

File: wordle-web/pages/game.vue [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+          <hr class="my-3" />
```
Comment (mmwoodfo) [inline wordle-web/pages/game.vue:19]: Consider using the `v-divider` component here: https://vuetifyjs.com/en/components/dividers/

**Comment by mmwoodfo** [comment]
Looks like your actions pipeline is failing on the deployment step. Did you set up an Aure website?

**Pull request closed without merging**